### DISCUSSION TITLE
Simulating a transmission line in transient mode

### DISCUSSION THREAD

**Discussion by oe1rsa** [ra3xdh/qucs_s#836] [Q&A] [answered]
I am trying to use a simple transmission line within a nonlinear circuit. However, first I tried to understand how transmission lines work in simulation and came up with the following basic setting:
![grafik](https://github.com/user-attachments/assets/db44efef-32ed-41a3-852a-dc570f86fd8a)
But I do not understand the output. I would have expected that the "out" is delayed by 1.5m/3e8ms-1 = 5ns. Apparently it is not. Can anyone please help me understand what I am doing wrong or am overlooking.

Ah, yes I use Qucsator mode and the messages reads:

```
Starting new simulation on Fr. 12. Juli 2024 at 09:24:27:274

creating netlist... done.
Starting /usr/local/bin/qucsator

project location: 
modules to load: 0
factorycreate.size() is 0
factorycreate has registered:
parsing netlist...
checking netlist...
netlist content
      1 TLIN instances
      1 R instances
      1 Pac instances
      1 TR instances
creating netlist...
NOTIFY: TR1: average time-step 3.111e-12, 39080 rejections
NOTIFY: TR1: average NR-iterations 4.43156, 0 non-convergences

Simulation ended on Fr. 12. Juli 2024 at 09:24:27:637
Bereit.

```

**Comment by ra3xdh** [comment, marked as answer]
Qucsator parses 1.5m as 1.5e-3 instead of 1.5 meter. The prefix has priority before the unit and the `m` is assigned for both milli and meter. Set the `L=1.5` and the schematic will work as expected. 
![image](https://github.com/user-attachments/assets/915b53b7-c6d8-440a-b097-b631f9a1311f)

**Reply by oe1rsa** [reply]
Thank you very much. Now that I read your answer I remember that I had seen it some time ago in the spice manual.
